src/components/ADempiere/Report/Data/DataReport.vue

```diff
@@ -597,6 +597,13 @@ export default defineComponent({
 </script>
 
 <style>
+.el-table .el-icon-arrow-right:before {
```

Comment:
By not having a different or unique parent class for this component, and the `style` tag not having the `scope` property, the style change may affect other tables where the `.el-icon-arrow-right` class is used, in case a report is loaded first and then a window, smart query or form with table is opened.


Please set a unique class to the root tag of the component and place the style tag as scss:
```html
<style lang=“scss”>
.unique-class-to-root {
  .el-table .el-icon-arrow-right-right:before {
    // code
  }
}
</style>
```

Or just add the `scope` property `<style scope>` but sometimes it generates unexpected behavior with classes that are global like `.el-table`.